adserver/tasks.py

```diff
@@ -604,6 +604,12 @@ def daily_update_publishers(day=None):
             )
 
 
+@app.task(time_limit=60 * 60 * 4)
+def daily_update_reports():
+    day, _ = get_day()
+    update_previous_day_reports(day)
```

Comment:
I don't quite understand why we have this function, instead of just calling `update_previous_day_reports` via celery? It seems like it's the same logic?

Reply:
The key difference is that `update_previous_day_reports` does the previous day while `daily_update_reports` does today. Should I just add a docstring?

Reply:
But the logic is the same? `start_date, _ = get_day(day)`